engine/src/cmd/role_bitmask.cpp

```diff
@@ -71,21 +71,21 @@ static vsUMap<string, int> rolemap;
 static vsUMap<int, string> irolemap;
 
 unsigned char InternalGetRole(const std::string &s) {
-    vsUMap<string, int>::const_iterator i = rolemap.find(strtoupper(s));
```

Comment:
This (all modifications to this file) should be in a separate commit, IMHO

Reply:
we take a more pragmatic approach.